File: RepoTransBench/target_projects/Python/Java/aadhithya_rajiniPP/src/test/java/com/rajinipp/original/RunnerTest.java
package com.rajinipp.original;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
import java.io.ByteArrayOutputStream;
import java.io.PrintStream;

public class RunnerTest {

    private void runWithStdoutCapture(String code, java.util.function.Consumer<String> assertion) {
        PrintStream oldOut = System.out;
        PrintStream oldErr = System.err;
        ByteArrayOutputStream baos = new ByteArrayOutputStream();
        System.setOut(new PrintStream(baos));
        try {
            com.rajinipp.Rajinipp.rpp.exec(code);
        } finally {
            System.out.flush();
            System.setOut(oldOut);
            System.setErr(oldErr);
        }
        assertion.accept(baos.toString().trim());
    }

    private String loadExample(String filename) {
        return com.rajinipp.util.ExampleLoader.loadExample(filename);
    }

    @Test
    void testExec() {
        String code = loadExample("hello_world.rpp");
        runWithStdoutCapture(code, (out) ->
                assertEquals("Hello, World!", out));
    }

    @Test
    void testEval() {
        double result = com.rajinipp.Rajinipp.rpp.eval("5+5;");
        assertEquals(10.0, result, 0.0001);
    }
}

File: RepoTransBench/target_projects/Python/Go/aadhithya_rajiniPP/tests/original/runner_test.go
package original

import (
	"testing"

	"yourmodule/tests/testutil"
)

func TestExec(t *testing.T) {
	output, _ := testutil.ExecAndCapture(testutil.GetFixturePath("hello_world.rpp"))
	if output != "Hello, World!" {
		t.Fatalf("Expected 'Hello, World!', got: %q", output)
	}
}

func TestEval(t *testing.T) {
	result := testutil.Eval("5+5;")
	if result != 10.0 {
		t.Fatalf("Expected result 10.0, got: %v", result)
	}
}